app/helpers/poem_helper.rb

```diff
@@ -1,8 +1,24 @@
 module PoemHelper
+  # TODO: 他の箇所でも使うようになったら(コメントとか)
+  #       ここじゃない場所に移した方が良さそう
   def markdown(text)
-    renderer = Redcarpet::Render::HTML.new(filter_html: true)
+    renderer = Redcarpet::Render::HTML.new
     extensions = {}
     markdown = Redcarpet::Markdown.new(renderer, extensions)
-    markdown.render(text).html_safe
+    markdown.render(emojify(text)).html_safe
+  end
+
+  def emojify(content)
+    return unless content.present?
+
+    h(content).to_str.gsub(/:([\w+-]+):/) do |match|
+      emoji_alias = Regexp.last_match(1)
+      emoji = Emoji.find_by_alias(emoji_alias)
+      if emoji
+        %(<img alt="#{emoji_alias}" src="#{image_path("emoji/#{emoji.image_filename}")}" class="emoji" />)
```

Comment:
__[rubocop]__
* Line is too long. [106/80] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)